Sources/Classes/PDFViewController.swift

```diff
@@ -84,8 +84,17 @@ public final class PDFViewController: UIViewController {
     /// UIBarButtonItem used to override the default action button
     fileprivate var actionButton: UIBarButtonItem?
     
+    /// Default background color applied both to the view and the collectionView.
+    fileprivate var defaultBackgroundColor: UIColor {
+        return UIColor(red: 236/255, green: 239/255, blue: 241/255, alpha: 1)
+    }
+    
     override public func viewDidLoad() {
         super.viewDidLoad()
+        
+        self.view.backgroundColor = defaultBackgroundColor
+        
+        collectionView?.backgroundColor = defaultBackgroundColor
```

Comment:
Hmm yeah I cannot understand either. I wonder if this is something left over from a previous version where this property was optional? As the compiler wouldn't complain about it being accessed this way. Its accessed from the ```viewDidLoad``` function, so I don't see a situation whereby this would not be set (as its an ```IBOutlet```).